src/store/transactions/post-conditions.hooks.ts

```diff
@@ -1,6 +1,23 @@
-import { postConditionModeState } from '@store/transactions/post-conditions';
 import { useAtomValue } from 'jotai/utils';
+import { addressToString, PostCondition, PostConditionType } from '@stacks/transactions';
+import { postConditionModeState } from '@store/transactions/post-conditions';
+import { fungibleTokenMetaDataState } from '@store/assets/fungible-tokens';
+import { useCurrentNetworkState } from '@store/network/networks.hooks';
 
 export const usePostConditionModeState = () => {
   return useAtomValue(postConditionModeState);
 };
+
+export const useAssetFromPostCondition = (pc: PostCondition) => {
+  if (pc.conditionType !== PostConditionType.Fungible) return;
```

Comment:
This should come after any hooks, to conditionally render hooks breaks the rules of hooks

Reply:
Yep yep, thanks for catching. 👍